lib/govuk_publishing_components/presenters/image_card_helper.rb

```diff
@@ -40,7 +40,20 @@ def image
       end
 
       def context
-        content_tag(:p, @context, class: "gem-c-image-card__context") if @context
```

Comment:
If we supply the content as a block we can remove the need to duplicate the content tag markup.

How about something like this:

```rb
def context
  return unless @context

  content_tag(:p, class: 'gem-c-image-card__context') do
    if @context[:date]
      date = content_tag(:time, l(@context[:date], format: '%e %B %Y'), datetime: @context[:date].iso8601)
      dash = content_tag(:span, ' — ', 'aria-hidden': true)

      if @context[:text]
        date.concat(dash).concat(@context[:text])
      else
        date
      end
    else
      @context[:text]
    end
  end
end
```

Reply:
@andrewgarner done 🙂